buildSrc/installerSigner.js

```diff
@@ -91,35 +101,64 @@ function signWithHSM(filePath, signatureFileName, dir) {
 	return path.join(dir, signatureFileName)
 }
 
-function signWithSelfSignedCertificate(filePath, signatureFileName, dir) {
-	const sigOutPath = path.join(dir, signatureFileName)
+/**
+ * takes a pem-encoded private key and returns the raw der-encoded data
+ * @param key {string} private key pem
+ * @returns {ArrayBuffer} raw binary der-encoded private key
+ */
+
+function pemToBinaryDer(key) {
+	const pemContentsB64 = key
+		.replace("-----BEGIN PRIVATE KEY-----", "")
+		.replace("-----END PRIVATE KEY-----", "")
+		.replace(/\s/g, '')
+	return base64ToUint8Array(pemContentsB64).buffer
+}
 
-	// copy cert to webserver so it can be easily put into a VM
-	fs.copyFileSync(path.join(process.env.DEBUG_SIGN, "ca.crt"), path.join(dir, "ca.crt"))
+/**
+ * import a key from a pem-string and import it as a WebCrypto CryptoKey
+ * that can be used for signing
+ * @param pem
+ * @returns {Promise<CryptoKey>}
+ */
+async function importPrivateKey(pem) {
+	return crypto.webcrypto.subtle.importKey(
```

Comment:
`webcrypto` was experimental until node 19.0.0, we still use 16.16. I think there are other APIs in node besides SubtleCrypto which can do it like this one? https://nodejs.org/api/crypto.html#cryptocreateprivatekeykey

Reply:
well that's kinda sad, should we just go with experimental API then?

Reply:
kind of a bummer that non-experimental webcrypto will hit node LTS in a year at the earliest, but since this is a build dependency and the signatures will be verified with the standard node crypto module I'm not too worried about using it...